files/jquery.photoswipe/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
```

Comment:
No tags here https://github.com/yaquawa/jquery.photoswipe/releases
the bot will fail